packages/aws-otel-collector/aws-otel-collector.spec

```diff
@@ -64,11 +66,15 @@ install -d %{buildroot}{%{_cross_bindir},%{_cross_fips_bindir}}
 install -p -m 0755 aws-otel-collector %{buildroot}%{_cross_bindir}
 install -p -m 0755 fips/aws-otel-collector %{buildroot}%{_cross_fips_bindir}
 
+%cross_install_sbom
+
 %files
 %{_cross_attribution_file}
 %{_cross_unitdir}/aws-otel-collector.service
 %{_cross_tmpfilesdir}/aws-otel-collector-tmpfiles.conf
 %{_cross_factorydir}%{_cross_sysconfdir}/aws-otel-collector.yaml
+%{_cross_sbom_package_dir}/%{name}-spdx.json
```

Comment:
It's going to be gnarly, but to avoid breaking compatibility with all existing specs when the macros to generate and install SBOM files are enabled by default, you could follow the pattern of `%_debuginfo_template` and auto-generate a "-sbom" subpackage which then automatically includes these files.

That subpackage could then add a `Supplements: %{name}` field to set a reverse dependency on the main package to trigger its installation.